src/+otp/+kuramotosivashinsky/jac.m

```diff
@@ -0,0 +1,5 @@
+function j = jac(~, u, D, L)
+
+j = -L*L - L - spdiags(u, 0, numel(u), numel(u))*D - spdiags(D*u, 0, numel(u), numel(u));
```

Comment:
and an indent  here

Reply:
We should probably precompute L*L to avoid matrix-matrix multiplications every Jac eval